client/dashboard/sites/settings-sftp-ssh/enable-sftp-card.tsx

```diff
@@ -0,0 +1,114 @@
+import { useMutation } from '@tanstack/react-query';
+import {
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	__experimentalText as Text,
+	Button,
+	Card,
+	CardBody,
+	ExternalLink,
+	Panel,
+	PanelBody,
+} from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { store as noticesStore } from '@wordpress/notices';
+import { siteSftpUsersCreateMutation } from '../../app/queries';
+
+const FILEZILLA_URL = 'https://filezilla-project.org/';
+
+export default function EnableSftpCard( {
+	siteSlug,
+	canUseSsh,
+}: {
+	siteSlug: string;
+	canUseSsh: boolean;
+} ) {
+	const mutation = useMutation( siteSftpUsersCreateMutation( siteSlug ) );
+	const { createSuccessNotice, createErrorNotice } = useDispatch( noticesStore );
+
+	const handleCreateCredentials = () => {
+		mutation.mutate( undefined, {
+			onSuccess: () => {
+				createSuccessNotice( __( 'Credentials have been successfully created.' ), {
+					type: 'snackbar',
+				} );
+			},
+			onError: () => {
+				createErrorNotice(
+					__(
+						'Sorry, we had a problem retrieving your sftp user details. Please refresh the page and try again.'
+					),
+					{
+						type: 'snackbar',
+					}
+				);
+			},
+		} );
+	};
+
+	return (
+		<Card>
+			<CardBody>
+				<Text variant="muted" lineHeight="20px" style={ { paddingBottom: '12px' } } as="p">
```

Comment:
Let’s add a `Get started with SFTP/SSH` heading here. It helps resolve an issue where's there's not enough top padding on the card.

![image](https://github.com/user-attachments/assets/0b9df5a9-1a58-45dc-a67f-e8c49a8ae480)